omaha_server/crash/forms.py

```diff
@@ -97,3 +98,23 @@ def clean_file(self):
         except:
             raise forms.ValidationError(u"The file contains invalid data.")
         return file
+
+
+class TextInputForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        field_name = kwargs.pop('field_name')
+        super(TextInputForm, self).__init__(*args, **kwargs)
+        self.fields[field_name] = forms.CharField(
+            widget=TextInput(attrs={'placeholder': 'Filter by ID'}),
+            label='',
+            required=False)
+
+    def is_valid(self):
+        valid = super(TextInputForm, self).is_valid()
+        if not valid:
+            return valid
+        _id = self.cleaned_data['id']
+        if not _id.isdigit():
```

Comment:
I think we need to change this to `if _id and _id.isdigit():`, otherwise the "ID should be integer" warning will be shown even if there is no any filtering at all